<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendMessageFrame</class>
 <widget class="QFrame" name="SendMessageFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Send To:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_addressEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="m_addressBookButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_pasteButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/paste</normaloff>:/icons/paste</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Encrypted message:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTextEdit" name="m_messageTextEdit">
     <property name="html">
      <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.Helvetica Neue DeskInterface'; font-size:13pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="m_addReplyToCheck">
       <property name="text">
        <string>Add &quot;Reply To&quot;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>5</number>
     </property>
     <item row="1" column="1">
      <widget class="QSlider" name="m_mixinSlider">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
       <property name="tracking">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBothSides</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Anonimity level:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="m_mixinEdit">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>298</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Fee:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="m_feeSpin">
       <property name="decimals">
        <number>8</number>
       </property>
       <property name="minimum">
        <double>1.001000000000000</double>
       </property>
       <property name="maximum">
        <double>1000000000000000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>1.001000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_sendButton">
       <property name="text">
        <string>Send</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/send</normaloff>:/icons/send</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_segButton">
       <property name="text">
        <string>Segment</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/send</normaloff>:/icons/send</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_tickerLabel">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_addressBookButton</sender>
   <signal>clicked()</signal>
   <receiver>SendMessageFrame</receiver>
   <slot>addressBookClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pasteButton</sender>
   <signal>clicked()</signal>
   <receiver>SendMessageFrame</receiver>
   <slot>pasteClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>746</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_sendButton</sender>
   <signal>clicked()</signal>
   <receiver>SendMessageFrame</receiver>
   <slot>sendClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_mixinSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SendMessageFrame</receiver>
   <slot>mixinValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_addressEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SendMessageFrame</receiver>
   <slot>addressEdited(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_segButton</sender>
   <signal>clicked()</signal>
   <receiver>SendMessageFrame</receiver>
   <slot>segClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>713</x>
     <y>552</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addressBookClicked()</slot>
  <slot>pasteClicked()</slot>
  <slot>sendClicked()</slot>
  <slot>mixinValueChanged(int)</slot>
  <slot>addressEdited(QString)</slot>
  <slot>segClicked()</slot>
 </slots>
</ui>
